linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_retention_flush/ansible/shared.yml

```diff
@@ -8,6 +8,7 @@
     dest: /etc/audit/auditd.conf
     regexp: '.*flush.*'
```

Comment:
Since you touch this file, can you improve `regexp` to be more specific as others? For example have `^\s` and `=` sign in expression?

Reply:
Done... also added missing variable which is odd that it was missing in the first place.

Reply:
Nice catch on this.